src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import BigJs from 'big.js/big';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = null;
+
+  switch (operation) {
+    case '+':
+      result = BigJs(numberOne).plus(BigJs(numberTwo)).toString();
```

Comment:
```js
BigJs(numberOne).op(BigJs(numberTwo)).toString();
```
Is actually repetitive. So I think you can declare two variables that will contain numberOne and numberTwo.
```js
const firstNumber = BigJs(numberOne);
const secondNumber = BigJs(numberTwo);
```
then you can use them without repetition.